crates/re_types/definitions/rerun/blueprint/components/background_3d_kind.fbs

```diff
@@ -0,0 +1,24 @@
+include "arrow/attributes.fbs";
+include "python/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/attributes.fbs";
+
+namespace rerun.blueprint.components;
+
+
+/// The type of the background in 3D Space Views.
+enum Background3DKind: byte (
+    "attr.rerun.scope": "blueprint",
+    "attr.docs.unreleased"
+) {
+    /// Gradient depending on the direction of the view, dark theme.
+    DirectionalGradientDark,
+
+    /// Gradient depending on the direction of the view, bright theme.
+    DirectionalGradientBright,
```

Comment:
Can we shorten these to `GradientDark/GradientBright` or `DarkGradient/BrightGradient`?